tests/test_httpdigitalout.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import requests
+
+from labgrid.driver import HttpDigitalOutputDriver
+from labgrid.resource import HttpDigitalOutput
+
+
+@pytest.fixture(scope="function")
+def mock_server(mocker):
+    state = '"Unknown"'
+
+    def request(method, url, data=None):
+        nonlocal state
+        state = data
+        return mocker.MagicMock()
+
+    def get(url):
+        r = mocker.MagicMock()
+        r.text = state
+        return r
+
+    mock_request = mocker.patch("requests.request")
+    mock_request.side_effect = request
+    mock_get = mocker.patch("requests.get")
+    mock_get.side_effect = get
+
+    return (mock_request, mock_get)
+
+
+@pytest.fixture(scope="function")
+def make_http_driver(request, target):
+    def _make_http_driver(with_tls, with_regex, separate_get, match_error):
```

Comment:
This doesn't need to be a fixture returning a function. Instead just have `_make_http_driver` as a helper function and pass in the target explicitly.

Reply:
Sounds good. I've updated the PR just now to do so.